examples/hello_LoRa_frame_port/hello_LoRa_frame_port.ino

```diff
@@ -0,0 +1,82 @@
+// Hello LoRa - ABP TTN Packet Sender (Multi-Channel)
+// Tutorial Link: https://learn.adafruit.com/the-things-network-for-feather/using-a-feather-32u4
+//
+// Adafruit invests time and resources providing this open source code.
+// Please support Adafruit and open source hardware by purchasing
+// products from Adafruit!
+//
+// Copyright 2015, 2016 Ideetron B.V.
+//
+// Modified by Brent Rubell for Adafruit Industries, 2018
+// Added Frame Port by clavisound, Aug 2019
+/************************** Configuration ***********************************/
+#include <TinyLoRa.h>
+#include <SPI.h>
+
+// Visit your thethingsnetwork.org device console
+// to create an account, or if you need your session keys.
+
+// Network Session Key (MSB)
+uint8_t NwkSkey[16] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+
+// Application Session Key (MSB)
+uint8_t AppSkey[16] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+
+// Device Address (MSB)
+uint8_t DevAddr[4] = { 0x00, 0x00, 0x00, 0x00 };
+
+/************************** Example Begins Here ***********************************/
+// Data Packet to Send to TTN
+unsigned char loraData[11] = {"hello LoRa"};
+
+// How many times data transfer should occur, in seconds
+const unsigned int sendInterval = 30;
+
+// In which port we send data? [TTN supports 0-255]
+uint8_t framePort = 1;
+
+// Pinout for Adafruit Feather 32u4 LoRa
+TinyLoRa lora = TinyLoRa(7, 8, 4);
+
+// Pinout for Adafruit Feather M0 LoRa
+//TinyLoRa lora = TinyLoRa(3, 8, 4);
+
+void setup()
+{
+  delay(2000);
+  Serial.begin(9600);
+  while (! Serial);
+  
+  // Initialize pin LED_BUILTIN as an output
+  pinMode(LED_BUILTIN, OUTPUT);
+  
+  // Initialize LoRa
+  Serial.print("Starting LoRa...");
+  // define multi-channel sending
+  lora.setChannel(MULTI);
+  // set datarate
+  lora.setDatarate(SF7BW125);
+  if(!lora.begin())
+  {
+    Serial.println("Failed");
+    Serial.println("Check your radio");
+    while(true);
+  }
+  Serial.println("OK");
+}
+
+void loop()
+{
+  Serial.println("Sending LoRa Data...");
+  lora.sendData(loraData, sizeof(loraData), lora.frameCounter, framePort); // If you ommit FramePort it's assumed #1
```

Comment:
Could you explain this comment? I'm not sure what you mean..
> // If you ommit FramePort it's assumed #1

Reply:
You described correctly the behavior in your first comment. Will make the changes.

In my testing, TTN also accepts port 0. Is out of spec?